src/lib/Response/ResetResponse.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Response;
+
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Reset response to tell the app to reload the location.
+ */
+class ResetResponse extends Response implements NoRenderResponse
```

Comment:
Wouldn't that be `ResetLocationResponse` ? The `App-Location` header makes it quite specific. But do we need it to be location specific @dpobel ? We may want to redirect from a tab to an entirely different part of the admin.